x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/index/IndexAuditTrail.java

```diff
@@ -147,7 +147,7 @@
     private static final String FORBIDDEN_INDEX_SETTING = "index.mapper.dynamic";
 
     private static final Setting<Settings> INDEX_SETTINGS =
-            Setting.groupSetting(setting("audit.index.settings.index."), Property.NodeScope);
+            Setting.groupSetting(setting("audit.index.settings."), Property.NodeScope);
```

Comment:
I don't think we should change this here.
Group Settings are pretty horrible, and we're actively trying to reduce the scope of them, but this is increasing the scope.
Allowing `settings.*` seems wrong to me. I'd rather leave the group as `settings.index.*` and then clean up the keys as needed.

Reply:
I have kept the narrower scope for the group settings and used `Settings.Builder#normalizePrefix` to prepend the "index." prefix.